pyscriptjs/src/stores.ts

```diff
@@ -5,19 +5,7 @@ import type { Runtime } from './runtime';
 import type { AppConfig } from './pyconfig';
 import { getLogger } from './logger';
```

Comment:
unrelated but the above two statements i.e.
- `import type { AppConfig } from './pyconfig';` and
- `import { getLogger } from './logger';`

they both can also be removed since they are not used anywhere else in the file.

Reply:
and now that you deleted the stores below, even the following can be removed:
- `import type { Runtime } from './runtime';`
- `import type { PyLoader } from './components/pyloader';`

Reply:
good catch!
Fixed by 7ddfb38